docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+# docker-compose --env-file=.env.local up
+
+version: "3.9"
+services:
+  nginx:
+    build:
+      context: nginx
+    ports:
+      - "80:80"
+      - "443:443"
+    depends_on:
+      - mission-control
```

Comment:
Proč je tady tahle dependence?

Reply:
Asi relikt z testování a přepisování, myslím, že to není potřeba, odeberu to.